db/migrate/20221031190501_devise_create_users.rb

```diff
@@ -6,6 +6,8 @@ def change
       ## Database authenticatable
       t.string :email,              null: false, default: ""
       t.string :encrypted_password, null: false, default: ""
+      # t.string :username,           null: false, default: ""
+      # t.string :role,               null: false, default: "member"
```

Comment:
be careful! remember it's not ideal to edit existing migrations! this migration file was edited, however the schema file did not get updated. `role` should definitely be removed as a column until a pair is ready to implement that feature. you can decide as a group if you want to keep `username`.

Changing this migration means you either have to rollback all of your migrations, then re `migrate` so that these updates are evaluated. Or, everyone in the group needs to drop their databases and re `migrate` so that the schema is rebuilt with this information.